content/_getting-started/first-flow.md

```diff
@@ -1,172 +1,123 @@
 
-We assume that this is your first experience with the {{site.data.tenant.name}}
-platform, so we explain every step in detail. Please follow these steps to create
-your first integration flow.
+## Configuring Trigger
 
-## Choosing the data source
+The first step of an Integration Flow is called an initial trigger. Its job is to trigger the running Flow execution. [Components](/integration-component) that have a trigger mode can be used for a trigger, for example: Node.js, Webhook Component, Simple Trigger, etc. For the first step, the list of available components is limited to those that have trigger mode. In this instruction we will use Simple Trigger as an example of the first step.  
 
-When you login first time you will have a similar view of the {{site.data.tenant.name}} platform.
+To start the creation of our Flow, go to the navigational menu, select your Workspace and either click "Add new flow" on the dashboard:
 
-![Start creating your first flow](/assets/img/getting-started/first-flow/getting-started-flow-001.png "Start creating your first flow")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_0.png)
 
-To start click on *+Add new flow* button to load the integration flow designer.
+**OR** go to *Flows* and click "Add New Flow":
 
-The designer helps building the flows from the reusable integration components
-such as `triggers` and `actions`. Every integration flow starts with a `trigger`
-component to retrieve data from the source. To start let us find the
-*Petstore component* in the list of integration components.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_1.png)
 
-![Select our Petstore component](/assets/img/getting-started/first-flow/getting-started-flow-002.png "Select our Petstore component")
+This brings you to *Designer* page. Let's start with naming your Flow, which is optional, and adding the initial trigger:
 
-For that purpose type `petstore` in the search bar and you will find it. Next,
-click on *Choose Petstore API (Node.js)* button.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_4.png)
 
-![Create the credentials to access](/assets/img/getting-started/first-flow/getting-started-flow-003.png "Create the credentials to access")
+The platform prompts you to setup the first step. First you may give the step a special name. Then you can select the Component you want to use, by using search bar, Component list, or load new Component from Catalog:
 
-Next, you will see the page *Choose Credentials* where you need to click on
-*+connect new credentials* link to create your first credentials. Let us fill-in
-an input form for these two fields:
-*   **Account name** - you can leave this unchanged or provide your own.
-*   **API key** - please use `{{site.data.tenant.petStoreApiKey}}` to get access to our [Petstore API]({{site.data.tenant.petStoreDocs}}).
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_5.png)
```

Comment:
This screenshot contains a possibility to `Load new Component from Catalog` which is a custom implementation of one of our customers. Our product is not delivered with that option. Please create a new screenshot.